src/widgets/training-program/ui/training-program.module.scss

```diff
@@ -0,0 +1,91 @@
+.training-program {
+  &.column-2 {
+    display: flex;
+    flex-direction: row;
+    gap: 100px;
+    align-items: center;
+    justify-content: flex-start;
+
+    .left {
+      display: flex;
+      flex: 1 1 55%;
+      flex-direction: column;
+      gap: 26px;
+
+      max-width: 55%;
+
+      & span {
+        font-weight: $font-weight-bold;
+      }
+
+      div {
+        padding-bottom: 18px;
```

Comment:
Do we really need padding here? I see that there is already a gap between the elements, so maybe we can do without padding-bottom?

Reply:
removed